### PR TITLE
Add bootrr-auto.sh

### PR DESCRIPTION
**Pull request by alimon**
Suitable for run bootrr automatically at init, in this case
after log into the systemd via profile.d.

Signed-off-by: Aníbal Limón <anibal.limon@linaro.org>

### CODE REVIEW

**Comment by alimon** [comment]
@ndechesne @andersson Related to https://github.com/ndechesne/meta-qcom/pull/92

File: Makefile [hunk 1]
```diff
@@ -37,4 +37,8 @@ $(foreach v,${HELPERS},$(eval $(call add-scripts,helpers,$v)))
 
 install: $(all-install)
 
+install_auto: init/bootrr-auto.sh
+	echo "INSTALL $?"
+	install -D -m 755 $? $(DESTDIR)/etc/profile.d/bootrr-auto.sh
```
Comment (andersson) [inline Makefile]: Shouldn't this be $(DESTDIR)$(prefix)/etc... ?

Comment (andersson) [reply Makefile]: @ndechesne pointed out that this should not be /usr, so we're good on this front. But we should get the bootrr-auto.sh into the cpio archive as well.

File: init/bootrr-auto.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+if $(grep -q bootrr-auto /proc/cmdline); then
+	for TEST in $(tr "\0" "\n" < /proc/device-tree/compatible); do
+		if [ -x "/usr/bin/${TEST}" ]; then
```
Comment (ndechesne) [inline init/bootrr-auto.sh]: /usr/bin here make the assumption that prefix was set to /usr. we need to adjust based on $prefix.

Comment (alimon) [reply init/bootrr-auto.sh]: @ndechesne @andersson I just added a replacement take into account DESTDIR and prefix in the file.

**Comment by gctucker** [comment]
@alimon Since #21 has been merged, this should either be refactored to add the feature in `/etc/profile.d/` or dropped if the current solution is good enough.

**Pull request closed without merging**